### PR TITLE
Waf policy ga

### PR DESCRIPTION
**Pull request by wurp**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by wurp** [comment]
Note that https://github.com/Azure/azure-cli/pull/9701 is a prerequisite for this PR. Tests won't pass without it.

**Comment by wurp** [comment]
nudge

File: src/front-door/azext_front_door/azext_metadata.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
     "azext.isPreview": true,
```
Comment (tjprescott) [inline src/front-door/azext_front_door/azext_metadata.json]: If you plan to GA both front-door and waf-policy, then remove this line. If not, then please add the following in commands.py:
```Python
with self.command_group('network front-door', is_preview=True) as g:
  pass
```

File: src/front-door/azext_front_door/commands.py [hunk 2]
```diff
@@ -125,17 +125,34 @@ def load_command_table(self, _):
...
+        g.custom_command('list', 'list_azure_managed_rule_set')
+
+    with self.command_group('network front-door waf-policy managed-rules override', waf_policy_sdk) as g:
+        g.custom_command('add', 'add_override_azure_managed_rule_set')
+        g.custom_command('remove', 'remove_override_azure_managed_rule_set')
+
+    with self.command_group('network front-door waf-policy managed-rule-definition', waf_policy_sdk) as g:
+        g.custom_command('list', 'list_managed_rules_definitions')
+
+    with self.command_group('network front-door waf-policy rule', waf_policy_sdk) as g:
```
Comment (tjprescott) [inline src/front-door/azext_front_door/commands.py]: You should be able to just put the `supports_local_cache` and `model_path` kwargs in the command group definition to eliminate the duplication. 

```Python
 with self.command_group('network front-door waf-policy rule', waf_policy_sdk, supports_local_cache=True, model_path="...") as g:
```

**Pull request merged**